app/controllers/generic_crud_controller.rb

```diff
@@ -12,7 +12,13 @@ def create
 
     if get_instance_variable.save
       flash[:success] =  t "#{controller_name}.create.success"
-      redirect_to  action: :index
+
+      if params[:save_and_add].present?
+        redirect_to action: :new
+      else
+        redirect_to  action: :index
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.